### PR TITLE
Add trino groups for ccx superset access

### PR DESCRIPTION
**Pull request by accorvin**

### CODE REVIEW

**Comment by accorvin** [comment]
@mbacovsky here's the PR we've been talking about. I'd appreciate a +1 from you to make sure everything looks right to you.

File: kfdefs/base/trino/trino-group-mapping.properties [hunk 1]
```diff
@@ -19,3 +19,20 @@ eda:eda-admin
...
+apac-cs-csm-auto:anhuang,hyyoon,jberry,lefu,paharris,pkang,prgandhi,tquill,wili
+apac-cs-cse-auto:aali,abelsare,cchiu,dforte,eulim,goda,hchadha,jberry,jkho,mfuruya,munna,rpiddapa,sjadhav,sokuda,tow,vbanduni,wili,yyin
+na-cs-csm-auto:alospino,dpathak,erdo,filizola,jberry,jnoll,jofowler,karwilli,kiduffy,kjavier,krohlfs,lhsu,maparker,mniedzin,pobrien,rbrodney,rwicker,tdunham
+na-cs-cse-auto:aackerma,alospino,amrogers,asweitze,bicampbe,cbales,cpalmer,csheppar,damartin,deborah,dobrien,doreid,dpierre,echeng,fali,fganai,gkochavi,glthomas,hle,imahmed,irot,jbasquil,jberry,jholtby,jmclain,joharvey,johender,johsulli,jpedone,jwinfiel,kbeavers,kesnyder,kfrances,kliedman,kparsons,mbauer,mello,mforbush,mhenson,mhernon,mpreddice,nlozitsk,omali,opeavler,pzeigler,rbackofe,regray,rhale,sajenkin,sajoshi,sanparke,sblake,sfuller,sothomps,sstroud,tbramer,tdunham,trsmith,tschaibl,twest
+na-ps-cs-cse-auto:bicampbe,glthomas,jberry,jpedone,jychrist,kliedman,kparsons,regray,sajenkin,sfuller,sothomps
+latam-cs-csm-auto:hsonego,jberry,lasilva,llozoya,rsartori
+ccx-pm:psilling,rvokal,tdosek
+assisted-lakers:agentil,eerez,lgamliel,milevy,odepaz,rfreiman,rpiccoli
```
Comment (mbacovsky) [inline kfdefs/base/trino/trino-group-mapping.properties:38]: This is duplicate of line 11

Comment (accorvin) [reply kfdefs/base/trino/trino-group-mapping.properties]: Good catch. Fixed.

**Comment by mbacovsky** [comment]
@accorvin very nice! LGTM from me :+1:

**Review by DharmitD — APPROVED** [review]
/approve

**Pull request merged**